scripts/generate-fixture-snapshots.sh

```
@@ -0,0 +1,413 @@
+
+    git -C "$repo_root" worktree remove --force "$src_worktree" >/dev/null 2>&1 || true
+    rm -rf "$tmpdir"
+    return 0
+}
+
+# diff_trees <a> <b>: emit a brief diff, or nothing if identical.
+# Returns 0 if identical, 1 if different.
+diff_trees() {
+    diff -rq "$1" "$2" 2>&1 || return 1
+    return 0
+}
```

Shellcheck (0.11.0)

[info] This function is never invoked. Check usage (or ignored if invoked indirectly).

(SC2329)